packages/runtimes/hmr/src/loaders/hmr-runtime.js

```diff
@@ -197,7 +197,8 @@ function reloadCSS() {
   cssTimeout = setTimeout(function() {
     var links = document.querySelectorAll('link[rel="stylesheet"]');
     for (var i = 0; i < links.length; i++) {
-      var absolute = /^https?:\/\//i.test(links[i].getAttribute('href'));
+      var href = links[i].getAttribute('href')
+      var absolute = /^https?:\/\//i.test(href) && href.indexOf(window.location.href) !== 0
```

Comment:
Shouldn't this rather be `href.indexOf(window.location.origin) !== 0`?

Reply:
Yes, it should, good catch. Updated and `yarn format`ed.